### ISSUE TITLE
 Unhandled Exception: PlatformException(Failed to run model, Interpreter busy, java.lang.RuntimeException: Interpreter busy

### ISSUE THREAD

**Issue by RoliGautam21** [shaqian/flutter_tflite#287]
i am getting this issue while using face mask detection model

**Comment by dequilla3** [comment]
I've encountered this exception too but I ignored it since it doesn't stop the runtime. You can view this repo to get some refs. https://github.com/dequilla3/object_detector_tflite.git

**Comment by abellilo** [comment]
I'm also having this same issue

**Comment by abellilo** [comment]
> i am getting this issue while using face mask detection model

I'm also having this issue. Have you been able to solve it

**Comment by abellilo** [comment]
> I've encountered this exception too but I ignored it since it doesn't stop the runtime. You can view this repo to get some refs. https://github.com/dequilla3/object_detector_tflite.git

Yeah true. Just noticed that too. Ran the model using a button and noticed i was still getting my recognitions
